src/main/java/ua/chernonog/onlinebookstore/dto/request/CreateBookRequestDto.java

```diff
@@ -1,15 +1,30 @@
 package ua.chernonog.onlinebookstore.dto.request;
 
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.URL;
 
 @Data
 public class CreateBookRequestDto {
     private Long id;
+    @NotEmpty(message = "Title cannot be empty")
     private String title;
+    @NotEmpty(message = "Author cannot be empty")
     private String author;
+    @NotEmpty(message = "ISBN cannot be empty")
+    @Pattern(regexp = "^(?:\\d{9}[\\dX]|97[89]\\d{10})$", message = "Invalid ISBN format")
```

Comment:
I believe that there are already exists some @ISDN validation annotation, please check it out